project/accounts/models.py

```diff
@@ -169,10 +169,10 @@ def resize_profile_image(self):
         """
         Resizes and crops the user uploaded image and creates a thumbnail version of it
         """
-        profile_image_field = self.profile_image
-        image_file = io.StringIO(profile_image_field.read())
-        profile_image = Image.open(image_file)
-        profile_image.load()
+        #profile_image_field = self.profile_image
+        #image_file = io.StringIO(profile_image_field.read())
+        profile_image = Image.open(self.profile_image)
+        #profile_image.load()
```

Comment:
Remove unused code.
```suggestion
        profile_image = Image.open(self.profile_image)
```

Reply:
Sure I will have this done by today

Reply:
To make it easy, you can just press the "commit suggestion" button and the change will automatically be applied to the branch.

Reply:
updated PR